ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEyNDk2MzQzMjYuMTc0NTk1OTg2OCJ9LCJhbm9uSWQiOiI2NGIxZTlhNDM4NGY5NTg3In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI3MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjQ2OTcwOTk3MjI1NjU5MDYiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558ea98c0cea <unknown>
#1 0x558ea93715f0 <unknown>
#2 0x558ea9357e86 <unknown>
#3 0x558ea935679a <unknown>
#4 0x558ea9356eff <unknown>
#5 0x558ea9356e54 <unknown>
#6 0x558ea93757ff <unknown>
#7 0x558ea941a9aa <unknown>
#8 0x558ea93e8662 <unknown>
#9 0x558ea940e660 <unknown>
#10 0x558ea93e8433 <unknown>
#11 0x558ea93b4ea3 <unknown>
#12 0x558ea93b5b01 <unknown>
#13 0x558ea9885b3b <unknown>
#14 0x558ea9889a21 <unknown>
#15 0x558ea986cc32 <unknown>
#16 0x558ea988a594 <unknown>
#17 0x558ea9850eef <unknown>
#18 0x558ea98aed98 <unknown>
#19 0x558ea98aef76 <unknown>
#20 0x558ea98bfb36 <unknown>
#21 0x7fce4f09caa4 <unknown>
#22 0x7fce4f129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTI0OTYzNDMyNi4xNzQ1OTU5ODY4In0sImFub25JZCI6IjY0YjFlOWE0Mzg0Zjk1ODcifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouYXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvY291cGUvY2xlL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjcxIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sIl90b29sc0V2ZW50TGlua0lkIjoiMzM3NTE2MDk0MTQ3NDI3NDUiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558ea98c0cea <unknown>
#1 0x558ea93715f0 <unknown>
#2 0x558ea9357e86 <unknown>
#3 0x558ea935679a <unknown>
#4 0x558ea9356eff <unknown>
#5 0x558ea9356e54 <unknown>
#6 0x558ea93757ff <unknown>
#7 0x558ea941a9aa <unknown>
#8 0x558ea93e8662 <unknown>
#9 0x558ea940e660 <unknown>
#10 0x558ea93e8433 <unknown>
#11 0x558ea93b4ea3 <unknown>
#12 0x558ea93b5b01 <unknown>
#13 0x558ea9885b3b <unknown>
#14 0x558ea9889a21 <unknown>
#15 0x558ea986cc32 <unknown>
#16 0x558ea988a594 <unknown>
#17 0x558ea9850eef <unknown>
#18 0x558ea98aed98 <unknown>
#19 0x558ea98aef76 <unknown>
#20 0x558ea98bfb36 <unknown>
#21 0x7fce4f09caa4 <unknown>
#22 0x7fce4f129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyMzYzIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJkZV9BVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI3MSJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTI0OTYzNDMyNi4xNzQ1OTU5ODY4In0sImFub25JZCI6IjY0YjFlOWE0Mzg0Zjk1ODcifSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6eyJQcm9kdWN0Ijp7Il9pZCI6IjIzNjMiLCJkaW1lbnNpb25zIjp7IkJhdW11c3RlclNob3J0IjpbIjIzNjMiXX19fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIzMzk3MDU1MjM0MjI2NjU4In0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558ea98c0cea <unknown>
#1 0x558ea93715f0 <unknown>
#2 0x558ea9357e86 <unknown>
#3 0x558ea935679a <unknown>
#4 0x558ea9356eff <unknown>
#5 0x558ea9356e54 <unknown>
#6 0x558ea93757ff <unknown>
#7 0x558ea941a9aa <unknown>
#8 0x558ea93e8662 <unknown>
#9 0x558ea940e660 <unknown>
#10 0x558ea93e8433 <unknown>
#11 0x558ea93b4ea3 <unknown>
#12 0x558ea93b5b01 <unknown>
#13 0x558ea9885b3b <unknown>
#14 0x558ea9889a21 <unknown>
#15 0x558ea986cc32 <unknown>
#16 0x558ea988a594 <unknown>
#17 0x558ea9850eef <unknown>
#18 0x558ea98aed98 <unknown>
#19 0x558ea98aef76 <unknown>
#20 0x558ea98bfb36 <unknown>
#21 0x7fce4f09caa4 <unknown>
#22 0x7fce4f129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL2NvdXBlL2NsZS9vdmVydmlldy5odG1sIiwidmVoaWNsZUNvZGUiOiJjMjM2IiwidmVoaWNsZUJNIjoiMjM2MyIsInZlaGljbGVQSUhpc3RvcnkiOiIyMzYzIiwidGltZXN0YW1wIjoxNzQ1OTU5ODcyNjQ3LCJleHBpcmF0aW9uIjoxMjB9fX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTI0OTYzNDMyNi4xNzQ1OTU5ODY4In0sImFub25JZCI6IjY0YjFlOWE0Mzg0Zjk1ODcifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouYXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvY291cGUvY2xlL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjcxIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI2NDMyNDM5MzIxNzIwNDA2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x558ea98c0cea <unknown>
#1 0x558ea93715f0 <unknown>
#2 0x558ea9357e86 <unknown>
#3 0x558ea935679a <unknown>
#4 0x558ea9356eff <unknown>
#5 0x558ea9356e54 <unknown>
#6 0x558ea93757ff <unknown>
#7 0x558ea941a9aa <unknown>
#8 0x558ea93e8662 <unknown>
#9 0x558ea940e660 <unknown>
#10 0x558ea93e8433 <unknown>
#11 0x558ea93b4ea3 <unknown>
#12 0x558ea93b5b01 <unknown>
#13 0x558ea9885b3b <unknown>
#14 0x558ea9889a21 <unknown>
#15 0x558ea986cc32 <unknown>
#16 0x558ea988a594 <unknown>
#17 0x558ea9850eef <unknown>
#18 0x558ea98aed98 <unknown>
#19 0x558ea98aef76 <unknown>
#20 0x558ea98bfb36 <unknown>
#21 0x7fce4f09caa4 <unknown>
#22 0x7fce4f129c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/BFV1_attempt_2.png'